Geometry/GEMGeometry/src/GEMEtaPartitionSpecs.cc

```diff
@@ -13,14 +13,17 @@ GEMEtaPartitionSpecs::GEMEtaPartitionSpecs(SubDetector rss, const std::string& n
     float r0 = h * (B + b) / (B - b);
     float striplength = h * 2;
     float strips = _p[3];
-    float pitch = (b + B) / strips;
+    float dphi = _p[5] * TMath::Pi() / 180.;
```

Comment:
use https://github.com/cms-sw/cmssw/blob/master/DataFormats/Math/interface/angle_units.h to convert radians to degrees, avoid TMath